cmd/pxe/Dockerfile

```diff
@@ -0,0 +1,40 @@
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2018 Red Hat, Inc.
+#
+
+FROM fedora:28
+
+MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
+
+RUN dnf -y install \
+  syslinux \
+  && dnf -y clean all
+
+
+# Create non-root user
+RUN useradd -u 1001 --create-home -s /bin/bash pxe
+WORKDIR /home/pxe
+
+RUN mkdir -p /images && chown pxe:pxe /images
+
+USER 1001
+
+
+COPY pxe /usr/bin/pxe
```

Comment:
The PXE binary talks to the cluster. It is the one which allows assigning configs based on labels. 

We could say that instead of allowing label matching, we only allow UUID matching and add files as config maps, but even then we would introduce the complete stack of a http server to our responsibility regarding to updating and maintaining it. We use identical libraries for http like for all our other KubeVirt services, so the request sounds strange to me.